packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -3,130 +3,187 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-panel-footer-padding: Specifies the padding of the component's footer.
- * @prop --calcite-panel-header-border-block-end: Specifies the component header's block end border.
+ * @prop --calcite-panel-background-color: Specifies the background color of the component.
+ * @prop --calcite-panel-border-color: Specifies the border color of the component.
+ * @prop --calcite-panel-description-text-color: Specifies the text color of the component's description.
+ * @prop --calcite-panel-fab-z-index: Specifies the component fab's z-index.
+ * @prop --calcite-panel-footer-background-color: Specifies the background color of the component's footer.
+ * @prop --calcite-panel-footer-padding: [Deprecated] Use `--calcite-panel-footer-space` instead. Specifies the padding of the component's footer.
+ * @prop --calcite-panel-footer-space: Specifies the spacing of the component's footer.
+ * @prop --calcite-panel-header-background-color: Specifies the component header's background color.
+ * @prop --calcite-panel-header-border-block-end: [Deprecated] Use `--calcite-panel-header-border-width` instead. Specifies the component header's block end border.
+ * @prop --calcite-panel-header-border-width: Specifies the component header's border width.
```

Comment:
Should this one be deprecated?

Reply:
It is new and replaces the old one> https://github.com/Esri/calcite-design-system/pull/8670#discussion_r1483734423

Reply:
Oh, gotcha. Is there a use case for making this thicker border or removing? Otherwise not sure we need this.

Reply:
The use case was for removing the border

Reply:
Cool, I guess my vote would be to not add this? Unless you know the exact use case for it, seems strange we'd allow this to be removed. Maybe from a case where the header-content slot is filled?

Reply:
Agreed, we can just not add it.